source/grid/grid_generator.cc

```diff
@@ -419,6 +419,109 @@ namespace GridGenerator
     hyper_rectangle (tria, p1, p2, colorize);
   }
 
+  template <int dim, int spacedim>
+  void
+  simplex(Triangulation<dim, spacedim> &tria,
+          const std::vector<Point<spacedim> > &vertices)
+  {
+    AssertDimension(vertices.size(), dim+1);
```

Comment:
You mention a requirement about ordering of the vertices. Is it possible to check this in here (or what happens if I fail to do so?)?